zkevm/tests/test_util.rs

```diff
@@ -20,14 +20,13 @@ pub fn parse_trace_path_from_mode(mode: &str) -> &'static str {
     let trace_path = match mode {
         "empty" => "./tests/traces/empty.json",
         "greeter" => "./tests/traces/greeter.json",
-        "multiple" => "./tests/traces/multiple-erc20.json",
-        "native" => "./tests/traces/native-transfer.json",
-        "single" => "./tests/traces/single-erc20.json",
-        "single_legacy" => "./tests/traces/single-erc20-legacy.json",
-        "dao" => "./tests/traces/dao.json",
-        "nft" => "./tests/traces/nft.json",
-        "sushi" => "./tests/traces/masterchef.json",
-        _ => "./tests/traces/multiple-erc20.json",
+        "single" => "./tests/traces/erc20/single.json",
+        "multiple" => "./tests/traces/erc20/multiple.json",
+        "native" => "./tests/traces/native_transfer.json",
+        "dao" => "./tests/traces/dao/propose.json",
```

Comment:
So here we change the test sample from including mutiple actions (propose and cancel) to single one. I am not sure if this degration is appropriate

Reply:
I think it's fine.